pages/home.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+
+"""Page model for home page."""
+
+from pypom import Page
+
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as expected
+
+
+class HomePage(Page):
+    """Page object model for home page."""
+
+    def wait_for_page_to_load(self):
+        self.wait.until(
+            lambda _: self.is_element_displayed(
+                By.CSS_SELECTOR, ".dropdown--profile__username"
+            )
+        )
+        return self
+
+    @property
+    def title(self):
+        """Return the page title."""
+        return self.wait.until(lambda s: self.selenium.title)
+
+    @property
+    def profile_dropdown(self):
+        """Return the profile dropdown element."""
+        element = self.wait.until(
+            expected.visibility_of_element_located(
+                (By.CSS_SELECTOR, ".dropdown--profile__username")
```

Comment:
There are two variations of this element locator in three places. I would suggest using class attributes for locators to reduce duplication.